extensions/che-theia-hosted-plugin-manager-extension/package.json

```diff
@@ -15,20 +15,36 @@
     "@eclipse-che/theia-remote-api": "^0.0.1"
   },
   "scripts": {
-    "prepare": "yarn clean && yarn build",
+    "prepare": "yarn clean && yarn build && yarn test",
```

Comment:
Tests should be skipped if ENV variable SKIP_TEST=true is defined

Reply:
Added check for the environment variable to all necessary packages